### PR TITLE
Support attachments

### PR DESCRIPTION
**Pull request by hfm**
sample screen shot when execute the following command:

```rb
Takosan.privmsg 'awesome comment and title with link', color: '#000', title: 'black title', title_link: 'http://example.com'
```

image

### CODE REVIEW

**Comment by hfm** [comment]
@hsbt Please review this PR.

Now I'd like to stop supporting Ruby 1.9.3 because this version is too old to maintain... (But keep supporting Ruby 2.0.0 because it's system ruby in CentOS7.)

File: test/takosan_test.rb [hunk 1]
```diff
@@ -24,10 +24,26 @@ def test_privmsg
...
+    Net::HTTP::Post.expects(:new).with('/privmsg').returns(req)
+
+    Takosan.privmsg('foo bar buzz', title: 'black', color: '#000')
+  end
+
   def test_rescue_timeout_error
     http = stub('http')
     http.stub_everything
-    http.stubs(:request).raises(TimeoutError, 'timeout error!')
+    http.stubs(:request).raises(Timeout::Error, 'timeout error!')
```
Comment (hsbt) [inline test/takosan_test.rb:46]: 👍

File: lib/takosan.rb [hunk 2]
```diff
@@ -9,8 +9,8 @@ module Takosan
 
   module_function
 
-  def privmsg(message)
-    request('/privmsg', { 'channel' => @@channel, 'message' => message, 'name' => @@name, 'icon' => @@icon })
+  def privmsg(message, **attachments)
```
Comment (hsbt) [inline lib/takosan.rb]: Why you use `attachments`? I felt this variable usage as `options`.

Comment (hfm) [reply lib/takosan.rb]: I intended to use it as Slack's [message attachments](https://api.slack.com/docs/message-attachments). However, it seems to confusing, so I change it to `options`.

Comment (hfm) [reply lib/takosan.rb]: fixed by 146204b

**Pull request merged**